go/vt/servenv/grpc_server.go

```diff
@@ -136,6 +150,8 @@ func RegisterGRPCFlags() {
 	// Note: We're using 4 MiB as default value because that's the default in the
 	// gRPC 1.0.0 Go server.
 	GRPCMaxMessageSize = flag.Int("grpc_max_message_size", 4*1024*1024, "Maximum allowed RPC message size. Larger messages will be rejected by gRPC with the error 'exceeding the max size'.")
+	// Default is effectively infinity, as defined in grpc.
+	GRPCMaxConnectionAge = flag.Duration("grpc_max_connection_age", time.Duration(math.MaxInt64), "Maximum age of a client connection before GoAway is sent.")
```

Comment:
`math.MaxInt64`: nice find. We should change other places in the code that have the number hard-coded.